pkg/scheduler/scheduler.go

```diff
@@ -183,6 +194,12 @@ func (s *Scheduler) schedule(ctx context.Context) wait.SpeedSignal {
 	log := ctrl.LoggerFrom(ctx).WithValues("schedulingCycle", s.schedulingCycle)
 	ctx = ctrl.LoggerInto(ctx, log)
 
+	snapshotOpts := make([]cache.SnapshotOption, 0)
```

Comment:
```suggestion
	var snapshotOpts []cache.SnapshotOption
```
non-blocking nit, but please update as you are on it, in golang slices are lazily initialized, so this is more idiomatic